software/minus-games.yml

```
@@ -0,0 +1,11 @@
+# software name
+name: "Minus Games"
+website_url: "https://accessory.github.io/minus_games_user_guide"
+source_code_url: "https://github.com/Accessory/minus_games"
+description: "Server and launcher combo that helps you sync game and save files across multiple devices."
```

```suggestion
description: "Sync games and save files across multiple devices."
```
Cleaner description, in my opinion